coding/cpp_style_guide.rst

```diff
@@ -793,10 +793,10 @@ Makes it clear what the unit should do and especially all the things it is not s
 
 .. _style-guide-cpp-3-32:
 
-3-32. Parameters in functions SHOULD be declared in order of output, input, default input.
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+3-32. Parameters in functions should be ordered to maximize consistency among overloads.
```

Comment:
I assume SHOULD was capitalized for a reason before?

Reply:
It's to indicate that it conforms to https://developer.lsst.io/coding/intro.html#stringency-levels  This SHOULD be maintained. 😄